src/tyr/route_serializer_osrm.cc

```diff
@@ -1567,24 +1560,26 @@ TEST(RouteSerializerOsrm, testAddsIncidents) {
 
   rapidjson::Document serialized_to_json;
   {
-    auto intersection_doc = json::Jmap();
-    auto inner_node = TripLeg_Node();
+    auto intersection_doc = json::map({});
```

Comment:
from here down is all just unit tests updated to test the serialization on the leg